quilt/tools/store.py

```diff
@@ -105,6 +109,17 @@ def save_df(self, df, name, path, ext, target):
         """
         raise NotImplementedError()
 
+    def save_file(self, srcfile, name, path, target):
+        """
+        Save a (raw) file to the store.
+        """
+        self._find_path_write()
+        filehash = digest_file(srcfile)
+        fullname = name.lstrip('/').replace('/', '.')
```

Comment:
What is `fullname` used for? That is, given that contents.json already has nested groups, I was assuming that storing the path wouldn't be necessary.